account_voucher_tax/account_voucher.py

```diff
@@ -469,16 +459,17 @@ def _preparate_move_line_tax(self, cr, uid, src_account_id,
 
         return [debit_line_vals, credit_line_vals]
 
-    def _get_base_amount_tax_secondary(self, cr, uid, line_tax,
-                                       amount_base_tax, reference_amount,
-                                       context=None):
+    def _get_base_amount_tax_secondary(
+            self, cr, uid, line_tax, amount_base_tax, reference_amount,
+            context=None):
         amount_base = 0
         tax_secondary = False
-        if line_tax and line_tax.tax_category_id\
-                and line_tax.tax_category_id.name in \
-                ('IVA', 'IVA-EXENTO', 'IVA-RET', 'IVA-PART'):
-            amount_base = line_tax.amount and\
-                reference_amount / line_tax.amount or amount_base_tax
+        if all([line_tax, line_tax.tax_category_id,
```

Comment:
The same here:
https://github.com/Vauxoo/odoo-mexico-v2/pull/390#discussion_r113048608